internal/legacy/keystore/keystore.go

```diff
@@ -476,6 +476,8 @@ func (net *netParams) ReadFrom(r io.Reader) (int64, error) {
 		*net = (netParams)(chaincfg.TestNet3Params)
 	case wire.SimNet:
 		*net = (netParams)(chaincfg.SimNetParams)
+	case chaincfg.SigNetParams.Net:
```

Comment:
We'll also want to handle this in a few more places in `waddrmgr` after the account import PRs.